client/src/app/site/pages/meetings/services/active-meeting.subscription.ts

```diff
@@ -51,6 +52,34 @@ export function getActiveMeetingSubscriptionConfig(id: Id, settingsKeys: string[
                     idField: `projector_ids`,
                     fieldset: [`name`],
                     follow: [{ idField: `current_projection_ids`, fieldset: [`content_object_id`] }]
+                },
+                {
+                    idField: `reference_projector_id`,
+                    fieldset: [...MEETING_ROUTING_FIELDS, `used_as_reference_projector_meeting_id`],
+                    follow: [
+                        {
+                            idField: `current_projection_ids`,
+                            fieldset: [...MEETING_ROUTING_FIELDS],
+                            follow: [
+                                {
+                                    idField: `content_object_id`,
+                                    fieldset: [...MEETING_ROUTING_FIELDS],
+                                    follow: [
+                                        {
+                                            idField: `list_of_speakers_id`,
+                                            fieldset: [...MEETING_ROUTING_FIELDS],
```

Comment:
Do you really need to request the sequential number and meeting id for all of these? If not, couldn't you just leave them out and not request any fields for the connective models at all?